client/galaxy/scripts/mvc/dataset/dataset-li.js

```diff
@@ -252,21 +254,30 @@ var DatasetListItemView = _super.extend(
          */
         _renderShowParamsButton: function() {
             // gen. safe to show in all cases
+            let self = this;
             return faIconButton({
                 title: _l("View details"),
                 classes: "params-btn",
-                href: this.model.urls.show_params,
-                target: this.linkTarget,
                 faIcon: "fa-info-circle",
                 onclick: function(ev) {
-                    if (Galaxy.frame && Galaxy.frame.active) {
+                    ev.preventDefault();
+                    let showParamsInstance = Vue.extend( ShowParams ),
+                        mountView = document.createElement( "div" );
+                    if ( Galaxy.frame && Galaxy.frame.active ) {
                         Galaxy.frame.add({
-                            title: _l("Dataset details"),
-                            url: this.href
+                            title: _l("Dataset details")
                         });
-                        ev.preventDefault();
-                        ev.stopPropagation();
+                        let $elFrame = $( ".corner.frame" ),
+                            $elLatestFrame = $( $elFrame[ $elFrame.length - 1 ] ).find( ".f-content" );
```

Comment:
We should rely on internal css classes of the scratchbook viewer.

Reply:
I will look into it... thanks!